src/javascripts/Views/addMastersModal.js

```diff
@@ -0,0 +1,31 @@
+ngapp.controller('addMastersModalController', function($scope, $timeout, errorService, modalService) {
```

Comment:
You appear to have injected `modalService` here, but haven't defined an `unfocusModal` function, though you do use one in your HTML template.  You should add that in here.

Reply:
Resolved.  :+1: